### PR TITLE
1013 Update buttons on edit page and 1015 Update "Link a Sub-Event" button

### LINKED ISSUE #1013: [FEATURE] Update buttons on the edit page.
Please change the style of add contact button based on the shared design.

### LINKED ISSUE #1015: [FEATURE] update the button of `link sub-event` in user event detail page.

### PR DESCRIPTION
**Pull request by clav1d**
## Description
_Please provide a summary of the pull request and the issue it fixes. Please add necessary details, context, dependencies, explanation of when review is needed (see next section), etc._

**Fixes #1013**
**Fixes #1015**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: static/main.css [hunk 1]
```diff
@@ -526,7 +526,7 @@ h5 {
 }
 
 .add-a-contact:hover, .add-a-contact:active {
-  background-color: #fff;
+  background-color: #e84a27;
```
[the remark this answers is not in the document]
Comment (minump) [reply static/main.css:529]: Maybe the "plus" sign color should be changed as the button color is orange on hover?
Screen Shot 2023-01-11 at 1 50 53 PM

[the remark this answers is not in the document]
Comment (minump) [reply static/main.css:529]: Same for the link sub-event button

**Review by minump — COMMENTED** [review]
some comments on button hover color

File: static/main.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (sandeep-ps) [reply static/main.css:530]: Thanks for your feedback, Minu. 

Jiayuan and I discussed these during today's Events Manager meeting. The hover color change may need designer input as it's an icon. I'm gathering these questions in today's meeting notes and will share them soon. Jiayuan will change the cursor pointer.

**Comment by bingzhang** [comment]
wait for GUI designer feedback.

**Comment by bingzhang** [comment]
shall we merge it and create another issue? @sandeep-ps @clav1d

**Review by bingzhang — APPROVED** [review]

**Pull request merged**